app/dashboard/views.py

```diff
@@ -503,6 +510,60 @@ def accept_bounty(request):
     return TemplateResponse(request, 'process_bounty.html', params)
 
 
+def contribute(request):
+    """Contribute to the bounty.
+
+    Args:
+        pk (int): The primary key of the bounty to be accepted.
+
+    Raises:
+        Http404: The exception is raised if no associated Bounty is found.
+
+    Returns:
+        TemplateResponse: The accept bounty view.
+
+    """
+    bounty = handle_bounty_views(request)
+
+    params = get_context(
+        ref_object=bounty,
+        user=request.user if request.user.is_authenticated else None,
+        confirm_time_minutes_target=confirm_time_minutes_target,
+        active='contribute_bounty',
+        title=_('Contribute'),
+    )
+    return TemplateResponse(request, 'contribute_bounty.html', params)
+
+
+def social_contribution(request):
+    """Social Contributuion to the bounty.
+
+    Args:
+        pk (int): The primary key of the bounty to be accepted.
+
+    Raises:
+        Http404: The exception is raised if no associated Bounty is found.
+
+    Returns:
+        TemplateResponse: The accept bounty view.
+
+    """
+    bounty = handle_bounty_views(request)
+    promo_text = str(_("Check out this bounty that pays out ")) + f"{bounty.get_value_true} {bounty.token_name} {bounty.url}"
```

Comment:
Can we opt for an f-string here opposed to forcing to `str` and concatenation? The f-string, in most cases, will still be faster than concat.

Reply:
the reason for the concat is that the `_` operator, in my experience, cannot handle the dynamic variables